### ISSUE TITLE
Packaging broken because of uint64 overflow in auditbeat

### ISSUE THREAD

**Issue by ruflin** [elastic/beats#5783]
Packaging for auditbeat currently fails with the following error:

```
23:56:44 /usr/local/osx-ndk-x86/bin/../SDK/MacOSX10.11.sdk/usr/include/dispatch/module.map:1:27: warning: unknown attribute 'extern_c' [-Wignored-attributes]
23:56:44 # github.com/elastic/beats/vendor/github.com/fsnotify/fsevents
23:56:44 vendor/github.com/fsnotify/fsevents/wrap.go:38: constant 36893488147419103231 overflows uint64
23:56:47 make[2]: *** [prepare-package-cgo] Error 2
23:56:47 make[2]: Leaving directory `/var/lib/jenkins/workspace/elastic+beats+master+package/src/github.com/elastic/beats/auditbeat'
23:56:47 make[1]: *** [package] Error 2
23:56:47 make[1]: Leaving directory `/var/lib/jenkins/workspace/elastic+beats+master+package/src/github.com/elastic/beats/auditbeat'
```

The culprit seems to be the following line: https://github.com/elastic/beats/blob/master/vendor/github.com/fsnotify/fsevents/wrap.go#L38

Strangely the dependency did not change in the last 5 months. I will test if I can reproduce it locally.

**Comment by ruflin** [comment]
@adriansr @andrewkroh The problem was introduced with this PR: https://github.com/elastic/beats/pull/5575 We had the package in beats before but it was not used. It started to be used in the above PR. 

Also locally on my machine `go build` returns the overflow issue. I think our CI did not catch this issue because currently it's not complied for OS X. I will open a PR for that.

**Comment by andrewkroh** [comment]
https://github.com/fsnotify/fsevents/issues/31

**Comment by ruflin** [comment]
Probably related to https://github.com/golang/go/issues/21708

**Comment by ruflin** [comment]
@andrewkroh @adriansr I'm reopening this issue as this was auto closed by https://github.com/elastic/fsevents/pull/2 I think merging the PR does not fix it yet as it first needs to make it into the vendor directory of beats.

**Comment by andrewkroh** [comment]
Fixed by #5866
